dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/IEC63406/Controls/BaseControls/CurrentLimitation.mo

```diff
@@ -0,0 +1,217 @@
+within Dynawo.Electrical.Controls.IEC.IEC63406.Controls.BaseControls;
+
+/*
+* Copyright (c) 2024, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model CurrentLimitation
+  extends Dynawo.Electrical.Controls.IEC.IEC61400.Parameters.QLimitParameters;
+
+  //Nominal parameter
+  parameter Types.ApparentPowerModule SNom "Nominal converter apparent power in MVA";
+
+  //Parameters
+  parameter Types.PerUnit IMaxPu "Maximum current" annotation(
+    Dialog(tab = "QControl"));
+  parameter Types.PerUnit QMaxUd "Maximum reactive power defined by users" annotation(
+    Dialog(tab = "QControl"));
+  parameter Types.PerUnit QMinUd "Minimum reactive power defined by users" annotation(
+    Dialog(tab = "QControl"));
+  parameter Boolean QLimFlag "0 to use the defined lookup tables, 1 to use the constant values" annotation(
+    Dialog(tab = "QControl"));
+  parameter Boolean PriorityFlag "0 for active current priority, 1 for reactive current priority" annotation(
+    Dialog(tab = "QControl"));
+
+  //Input variables
+  Modelica.Blocks.Interfaces.RealInput iPcmdPu(start = -P0Pu * SystemBase.SnRef / (SNom * U0Pu)) "Active current command" annotation(
+    Placement(visible = true, transformation(origin = {-140, 100}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-140, 100}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput uMeasPu(start = U0Pu) annotation(
```

Comment:
Could you add parameter descriptions for all parameters and initial parameters, input, and output variables, in all new added models?